tests/run/ctypedef_bint.pyx

```diff
@@ -0,0 +1,69 @@
+from cython cimport typeof
+
+ctypedef bint mybool
+
+cdef mybool mybul = True
+cdef bint bul = True
+cdef int num = 42
+
+
+def CondExprNode_to_obj(test):
+    """
+    >>> CondExprNode_to_obj(True)
+    True
+    2
+    >>> CondExprNode_to_obj(False)
+    True
+    84
+    """
+
+    print(typeof(mybul if test else num) == typeof(bul if test else num) == typeof(object))
```

Comment:
If such a test fails, it will unhelpfully print `False`, without any indication for the reason of the failure.
Just print all three values and let the doctest do the validation.